lib/modules/showImages.js

```diff
@@ -428,7 +428,6 @@ module.go = () => {
 	}
 
 	findAllImages(document.body);
-	document.addEventListener('dragstart', () => false);
```

Comment:
lol this has been around forever:
https://github.com/honestbleeps/Reddit-Enhancement-Suite/blame/3c442d75707318037f0f606341b29d7c6e56e3a1/lib/modules/showImages.js#L249
and I cleaned it up twice and never noticed it:
https://github.com/honestbleeps/Reddit-Enhancement-Suite/commit/6a9ce6a3379df0d92ef12ec8e4b60d463c9809bf#diff-81e82c385dc9912064f4bc5e91b47e58L249
https://github.com/honestbleeps/Reddit-Enhancement-Suite/commit/9e80b5731ad37a527dc5179fbc1944a334426bda#diff-81e82c385dc9912064f4bc5e91b47e58L356